logger/umlogger.go

```diff
@@ -0,0 +1,141 @@
+// Package logger provides implementetion of writing Log messages to Graylog, file, and stdout.
+// Supports log levels and destination.
+package logger
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	graylog "gopkg.in/gemnasium/logrus-graylog-hook.v2"
+)
+
+// ErrFailedToConfigureLog is the error returned when configuring failed for some reasons
+var ErrFailedToConfigureLog = errors.New("Failed to init log: failed to configure ")
+
+// StorageConfig contains fileds used in Connect for DSN
+type LogConfig struct {
+	Host       string
+	Port       string
+	PassSecret string
+	PassSHA2   string
+	Output     string
+	Level      string
+	Type       string
+}
+type loggerKeyType int
+
+const loggerKey loggerKeyType = iota
```

Comment:
iota does not work out of block of constants, if you need to set zero to constant - just set zero.

Reply:
fixed, removing it